### PR TITLE
Minor fixes in scripting_first_script.rst

### PR DESCRIPTION
**Pull request by theDrake**
Changed "by default" to "at their default values" and "calling the rotated() method on any Vector2" to "calling the Vector2 method rotated()" (in context, these changes are correct). Also added a missing comma.

### CODE REVIEW

File: getting_started/step_by_step/scripting_first_script.rst [hunk 1]
```diff
@@ -85,7 +85,7 @@ The Attach Node Script window appears. It allows you to select the script's
 language and file path, among other options.
 
 Change the Template field from "Node: Default" to "Object: Empty" to start with a clean file. Leave the
-other options by default and click the Create button to create the script.
+other options at their default values and click the Create button to create the script.
```
Comment (AThousandShips) [inline getting_started/step_by_step/scripting_first_script.rst:88]:
```suggestion
other options with their default values and click the Create button to create the script.
```
"at" doesn't make sense in English here, "with" should be better IMO

File: getting_started/step_by_step/scripting_first_script.rst [hunk 2]
```diff
@@ -283,7 +283,7 @@ defines a local variable: it only exists within the function's scope.
 We define a local variable named ``velocity``, a 2D vector representing both a
 direction and a speed. To make the node move forward, we start from the Vector2
 class's constant ``Vector2.UP``, a vector pointing up, and rotate it by calling the
-``rotated()`` method on any ``Vector2``. This expression, ``Vector2.UP.rotated(rotation)``,
+Vector2 method ``rotated()``. This expression, ``Vector2.UP.rotated(rotation)``,
```
Comment (AThousandShips) [inline getting_started/step_by_step/scripting_first_script.rst:286]:
```suggestion
``Vector2`` method ``rotated()``. This expression, ``Vector2.UP.rotated(rotation)``,
```

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks! In the future please keep your PRs to one commit. I squashed and merged this one but we prefer if contributors keep it to one commit themselves. If you don't know how to do that we have a guide [here](https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html#modifying-a-pull-request).

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.1.

**Pull request merged**